install.sh

```diff
@@ -829,8 +829,17 @@ do_unmount_and_remove() {
     set -x
 }
 
+# Clean server directory and preserve db directory
+find /var/lib/rancher/k3s/server ! -name "db" -mindepth 1 -maxdepth 1 -exec rm -rf {} +
```

Comment:
I am a little concerned that this introduces a new dependency into the uninstall script - at the moment we do not require findutils be installed on the host. 

Would `for dir in $(ls -d /var/lib/rancher/k3s/server/*/); do rm -rf "$dir"; done` do the same thing without requiring any new tools?

Reply:
Actually I'm still concerned that this is too prescriptive. When I pointed out server/db and agent/containerd as examples of things that users might want to mount, this was not expected to be an exhaustive list.

Can you think of a good way to effectively clean up these directories while leaving behind any **arbitrary** mount points the user may create under the agent and server dirs?

Reply:
I did a recursive solution while using the `/proc/mounts` the main idea with this new solution is that we will not delete any `dir` that has mounted `dir` inside or it is a mounted `dir` itself and will clean everything inside.

btw tks for the advice of using the `/proc/mounts` @brandond, my first solution before setting the mount points hardcoded was to use a recursive strategy too, but it began to be a rocket science.